build.sbt

```diff
@@ -38,9 +38,11 @@ lazy val microservice = (project in file("."))
 val httpComponentsVersion = "4.5.11"
 
 val compileDependencies = Seq(
-  "uk.gov.hmrc"                   %% "govuk-template"               % "5.54.0-play-26",
-  "uk.gov.hmrc"                   %% "play-ui"                      % "8.11.0-play-26",
   "uk.gov.hmrc"                   %% "bootstrap-frontend-play-26"   % "2.24.0",
+  "uk.gov.hmrc"                   %% "play-ui"                      % "8.14.0-play-26",
+  "uk.gov.hmrc"                   %% "play-frontend-govuk"          % "0.53.0-play-26",
+  "uk.gov.hmrc"                   %% "play-frontend-hmrc"           % "0.26.0-play-26",
```

Comment:
We do not need this one I believe.

Reply:
You're right we don't need play-frontend-govuk, but the other two we do